ui_components/src/travel/Travel.js

```diff
@@ -463,13 +463,12 @@ function Travel({
                             )}
                             {((mapData && mapData.operations && !mapData.operation_type) && typeof mapData.operations_type === 'undefined') && (
                                 Object.entries(mapData.operations).map(([key, value], index) => (
-                                    <button key={index} onClick={() => BeginOperation(key)} className='button'>
-                                        {`${value}`}
+                                    <button key={index} onClick={() => BeginOperation(key)} className='button' dangerouslySetInnerHTML={{ __html: value }}>
```

Comment:
Double-checking: Where on the backend is this value set, and is it user-generated at all?

Reply:
Not user generated, this comes from WarManager::getValidOperations
```$health_gain = floor($this->user->level / 2);
$valid_operations[Operation::OPERATION_REINFORCE] .= "<br><span class='reinforce_button_text'>{$health_gain} health</span>";```